Places: `findPlaceFromText` photo typing has a typo

#### Environment details

1. Specify the API at the beginning of the title (for example, "Places: ...")
2. OS type and version: Linux 5.4.1
3. Library version and other environment information: `"@googlemaps/google-maps-services-js": "2.0.2"`

#### Steps to reproduce

1. Call `findPlaceFromText` adding `photos` as `fields`
2. Try to access the `photo_reference` property.
3. Ts type check will fail

Manually calling to `https://maps.googleapis.com/maps/api/place/findplacefromtext/json` gives the following output:

```json
{
   "candidates": [
      {
         "photos": [
            {
               "height": 4032,
               "html_attributions": [
                  "xxxx"
               ],
               "photo_reference": "xxxx",
               "width": 3024
            }
         ]
      }
   ],
   "status": "OK"
}
```

(Just to highlight, the property is called `photo_reference` not [`photoreference`](https://github.com/googlemaps/google-maps-services-js/blob/master/src/common.ts#L75))

#### Code example

```ts
const client = new Client({});
const { data: { candidates } } = await client.findPlaceFromText({
  params: {
    input,
    inputtype: PlaceInputType.textQuery,
    language: 'en',
    fields: ['photos'],
    key: googleMapsServerApiKey,
  },
});

if (candidates && candidates.length) {
  const [{ photos }] = candidates;

  if (photos && photos.length) {
    const [{ photo_reference }] = photos;
    // Do something with photo_reference
  }
}
```

This does compile if I change `photo_reference` to `photoreference`, but the value of `photoreference` is always `undefined`.

#### Stack trace
```
error TS2339: Property 'photo_reference' does not exist on type 'PlacePhoto'.
103       const [{ photo_reference }] = photos;
```

--- a/src/common.ts
+++ b/src/common.ts
@@ -72,7 +72,7 @@ export type Status =
 
 export interface PlacePhoto {
   /** a string used to identify the photo when you perform a Photo request. */
-  photoreference: string;
+  photo_reference: string;
   /** the maximum height of the image. */
   height: number;
   /** the maximum width of the image. */

